.expeditor/update_coverage.sh

```diff
@@ -10,8 +10,8 @@ COVERAGE_DATE="$(date +"%Y-%m-%dT%H:%M:%S")"
 #buildkite-agent artifact download "coverage/*" --build XXX-XXX-XXX(?)
 hab studio run "source .studiorc && code_coverage"
 
-aws --profile chef-cd s3 sync coverage "s3://chef-workstation-coverage/$BUILDKITE_PIPELINE_SLUG/current"
-aws --profile chef-cd s3 sync coverage "s3://chef-workstation-coverage/$BUILDKITE_PIPELINE_SLUG/$COVERAGE_DATE"
+aws --profile chef-cd s3 sync coverage "s3://chef-workstation-coverage/chef-chef-analyze-master-code-coverage/current"
```

Comment:
I bet there is an expeditor variable we can use here so this script becomes more portable and easier to apply to other projects

Reply:
https://expeditor.chef.io/docs/reference/workloads/#environment-variables
```
EXPEDITOR_SCM_SETTINGS_REPO=chef/example
EXPEDITOR_SCM_SETTINGS_BRANCH=master
```